github.js

```diff
@@ -35,16 +34,18 @@ async function constructPrMsg(sg, branch) {
  */
 function constructTrainNavigation(branchToPrDict, currentBranch, combinedBranch) {
   let contents = '<pr-train-toc>\n\n';
-  let tableData = [['', 'PR', 'Description']];
+  let prList = [];
   Object.keys(branchToPrDict).forEach((branch) => {
-    const maybeHandRight = branch === currentBranch ? '👉 ' : ' ';
+    const maybeHandRight = branch === currentBranch ? ' 👈' : ' ';
     const combinedInfo = branch === combinedBranch ? ' **[combined branch]** ' : ' ';
-    const prTitle = branchToPrDict[branch].title.trim();
     const prNumber = `#${branchToPrDict[branch].pr}`;
-    const prInfo = `${combinedInfo}${prTitle}`.trim();
-    tableData.push([maybeHandRight, prNumber, prInfo]);
+    const prInfoHtml = `<li>${combinedInfo}${prNumber}${maybeHandRight}</li>`
```

Comment:
OOC, would this work equally well as just
```
1. ${combinedInfo}${prNumber}${maybeHandRight}
```
i.e., could we simplify and avoid all the html markup?

Reply:
(Numbered list in markdown can just have all lines start with `1.` and the markdown processor will number them for us.)

Reply:
nice one. I didn't know all markdown processors supported that feature. Changed it and tested it out, looks a lot simpler now.